lib/info.js

```diff
@@ -18,6 +18,10 @@ var KEYS_TO_SPLIT = [
   'fexp',
   'watermark'
 ];
+var CONTENT_WARNINGS = new Set([
+  'Content Warning',
+  'Inhaltswarnung'
```

Comment:
Guess i missed out deleting that again. Was a little fix 'cause the test's kept failing